tools/ci_build/github/azure-pipelines/win-gpu-cuda-11-pipeline.yml

```diff
@@ -13,9 +13,10 @@ jobs:
     MsbuildArguments: '-detailedsummary -maxcpucount -consoleloggerparameters:PerformanceSummary'
```

Comment:
it's a little confusing that the yml file named win-gpu-cuda-11 is now switched to using cuda 10.2

Reply:
Yes, but if I change the file name now, I can't validate my changes in this file. So I plan to do it in two steps. First, make the content correct, second, rename the files.

Reply:
i see. understood.